LuaMenu/widgets/api_analytics.lua

```diff
@@ -139,11 +139,17 @@ end
 
 
 local function SocketConnect(host, port)
+	if client then
+		client:close()
+		client = nil
```

Comment:
Outdated version of patch? The upstream version doesn't have the redundant `= nil` here.

Reply:
Good catch :D, but we're upstream on this PR XD since I did it here first, I had to redo the patch for the other lobbys since this is not a fork I couldn't cherry pick, I guess I was more careful with that.